client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dashboard-v2.tsx

```diff
@@ -79,8 +82,8 @@ export default function SitesDashboardV2() {
 
 	const { data, isError, isLoading, refetch } = useFetchDashboardSites(
 		isPartnerOAuthTokenLoaded,
-		sitesViewState.search,
-		sitesViewState.page,
+		search,
+		currentPage,
```

Comment:
```suggestion
		sitesViewState.search,
		sitesViewState.page,
```

Reply:
In this case some components we are reusing will break, because they rely on using that context.

For instance: https://github.com/Automattic/wp-calypso/blob/bab4adcaa76dbbfaa2c0d78f2a93b37d409edbcb/client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-set-favorite/index.tsx#L28

If we change these keys the UI will stop updating as needed (e.g. when a user clicks to favorite a site), because we are not using the same keys as the components.

I will open an issue to address this, but I don't know how to go forward with this PR: do we make your suggested change and leave favorites (and possibly some other component) broken or do we leave as it is and check on how to use the context all components are using in a future PR?

Reply:
We have to use the state with the search and page value to fetch sites based on the search text. If not, the search functionality will stop working.

It seems that favorite stars click works partially, it gets the change and stores the site as favorite, or unset it, but it doesn't get the UI update because of what you say. But I'm not sure why not, since search and page doesn't seem required by favorite stuff.

Reply:
I think we could update the context, making the SitesOverview context as a state with useState, and then updating it from the SitesDashboardV2